doc/single-shortest-path.adoc

```diff
@@ -14,6 +14,34 @@ _All Pairs Shortest Path_ on the other hand calculates a shortest path forest co
 
 [source,cypher]
 ----
+MATCH (start:Node{type:'start'}), (end:Node{type:'end'})
+CALL algo.shortestPath.stream(start, end, 'cost') 
+YIELD nodeId, cost
+                       
+----
+
+
+[source,cypher]
+----
+MATCH (start:Node{type:'start'}), (end:Node{type:'end'})
+CALL algo.shortestPath(start, end, 'cost') 
+YIELD loadDuration, evalDuration, nodeCount, totalCost
+
+----
+
+[source,cypher]
+----
+MATCH(n:Node {name:'s'}) WITH n CALL algo.deltaStepping.stream(n, 'cost', 3.0)
```

Comment:
@mknblch should we expose implementation details in the procedure name? Can we find the better "common" algorithm name that's covered here?